bigchaindb/util.py

```diff
@@ -501,6 +501,26 @@ def get_hash_data(transaction):
     return crypto.hash_data(serialize(tx))
 
 
+def verify_vote_signature(signed_vote):
+    """Verify the signature of a vote
+
+    A valid vote should have been signed `current_owner` corresponding private key.
+
+    Args:
+        signed_vote (dict): a vote with the `signature` included.
+
+    Returns:
+        bool: True if the signature is correct, False otherwise.
+    """
+
+    data = signed_vote.copy()
+
+    signature = data.pop('signature')
+    public_key_base58 = signed_vote['node_pubkey']
```

Comment:
in the latest code we use `sk` and `vk`  (signing key/verifying key) as it stands closer to the crypto jargon

so `public_key_base58 => vk_base58`